go/Cargo.toml

```diff
@@ -10,8 +10,8 @@ crate-type = ["cdylib"]
 
 [dependencies]
 redis = { path = "../submodules/redis-rs/redis", features = ["aio", "tokio-comp", "tls", "tokio-native-tls-comp", "tls-rustls-insecure"] }
-glide-core = { path = "../glide-core" }
-tokio = { version = "^1", features = ["rt", "macros", "rt-multi-thread", "time"] }
+glide-core = { path = "../glide-core", features = ["socket-layer"] }
```

Comment:
:(

Reply:
Once we move away from protobuf, we can revert this.